rust-prog-mode.el

```diff
@@ -1457,7 +1574,7 @@ whichever comes first."
   (setq-local syntax-propertize-function #'rust-syntax-propertize)
 
   ;; Indentation
-  (setq-local indent-line-function 'rust-mode-indent-line)
+  (setq-local indent-line-function 'rust-doc-indent-line)
```

Comment:
Let's keep this variable to `rust-mode-indent-line` and tweak the functionality in that function. 🤔